client/src/theme/Components/backend/_drawer.scss

```diff
@@ -120,4 +120,9 @@
   .list-pagination-primary {
     @include constrainedPaginationResponsive;
   }
+
+  // full width selects
+  .form-secondary .form-select {
```

Comment:
Selects are full width in the backend drawers in the comp.  Let me know we were ok with them staying auto instead.